docs/reference/release-notes/highlights.asciidoc

```diff
@@ -58,6 +58,28 @@ Need a new enrollment token? Use the
 {ref}/create-enrollment-token.html[`elasticsearch-create-enrollment-token`] 
 tool to create enrollment tokens for {es} nodes and {kib} instances.
 
+[discrete]
+=== Better protection for system indices
```

Comment:
Suggestions for a better heading are appreciated.

Reply:
I'm fine with that heading, but here are alternatives for your consideration:

* Enhanced protection for system indices
* System indices are no longer directly accessible
* Safeguards for system indices